packages/pegasus/src/pegasus.js

```diff
@@ -14,7 +14,7 @@ import '@agoric/zoe/exported.js';
 
 import '../exported.js';
 import { IBCSourceTraceDenomTransformer } from './ibc-trace.js';
-import { ICS20TransferProtocol } from './ics20.js';
+import { DEFAULT_SENDER_ADDRESS, ICS20TransferProtocol } from './ics20.js';
```

Comment:
I'd prefer to avoid this import.

```suggestion
import { ICS20TransferProtocol } from './ics20.js';
```